src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
-export interface Figure {
+type Colors = 'red' | 'green' | 'blue';
 
+enum Shapes {
+  triangle = 'triangle',
```

Comment:
By the convention enum keys always start with the capital letter
```suggestion
  Triangle = 'triangle',
```

Reply:
+